presenters/dataPresenters.js

```diff
@@ -0,0 +1,42 @@
+const {formatDistanceToNow} = require('date-fns');
+
+function notesPresenter(interaction, notes) {
+  const formattedNotes = notes.reduce((validNotes, currentNote) => {
+    if (currentNote.valid) {
+      validNotes.push({
+        name: `ID: ${currentNote.id}   ${formatDistanceToNow(
+          currentNote.timestamp
+        )} ago`,
+        value: `${interaction.guild.members.cache.get(
+          currentNote.moderator
```

Comment:
Instead of `${interaction.guild.members.cache.get(currentNote.moderator)}`, we can use `<@${currentNote.moderator}>` since the purpose of this is just to display the mention and not obtain the GuildMember object for later use. Though if we want to use the former for consistency with how we usually fetch GuildMember objects so we can use them later, I'm good with that.

Reply:
This is a good point, and it also insulates us against errors if a moderator ends up leaving the server. Using `<@${currentNote.moderator}>` on a mod who has left will just translate to `@Unknown user` or whatever Discord uses to indicate a user who is no longer in the server 👍

Reply:
I presume you mean `${interaction.guild.members.cache.get(currentNote.moderator)}` would translate to `@Unknown user`?

Reply:
No, I was saying it the way I wrote it.  I don't remember if `guild.members.cache.get` every throws an error, but using `<@userId>` never does.  It will translate into the user's mention if they are in the server and fall back on `@Unknown User` if they are not.